spring-kafka/src/main/java/org/springframework/kafka/support/micrometer/KafkaListenerObservation.java

```diff
@@ -42,19 +49,32 @@ public Class<? extends ObservationConvention<? extends Context>> getDefaultConve
 		}
 
 		@Override
+		@NonNull
 		public String getPrefix() {
 			return "spring.kafka.listener";
 		}
 
 		@Override
+		@NonNull
 		public KeyName[] getLowCardinalityKeyNames() {
 			return ListenerLowCardinalityTags.values();
 		}
 
+		@Override
+		@NonNull
+		public KeyName[] getHighCardinalityKeyNames() {
+			return ListenerHighCardinalityTags.values();
+		}
+
 	};
 
 	/**
 	 * Low cardinality tags.
+	 *
+	 * @author Christian Mergenthaler
+	 * @author Wang Zhiyang
+	 *
+	 * @since 3.2
```

Comment:
This inner class was there before, so this `@since` is misleading.
Plus we prefer to have `@author` only on top level class, kinda one set per Java file 😄

Reply:
Sorry for this misleading, fix it. 😸